### PR TITLE
replace check for platform with check for platform_family

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

**Comment by vuntz** [comment]
@jsuchome see https://github.com/jsuchome/crowbar-ha/pull/1 for some suggestions

**Comment by vuntz** [comment]
+1 you might want to squash or drop the merge commit, though)

**Comment by vuntz** [comment]
(still +1)

**Comment by aplanas** [comment]
+1 Lets wait for the gating

File: chef/cookbooks/corosync/recipes/config.rb [hunk 1]
```diff
@@ -39,10 +39,10 @@
 end
 
 template "/etc/corosync/corosync.conf" do
-  if node.platform == "suse" && node.platform_version.to_f >= 12.0
-    source "corosync.conf.v2.erb"
-  else
+  if node[:platform_family] == "suse" && node[:platform_version].to_f < 12.0
```
Comment (vuntz) [inline chef/cookbooks/corosync/recipes/config.rb]: Actually, this one should stay `platform`.

File: chef/cookbooks/pacemaker/recipes/default.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@
   package pkg
 end
 
-unless node.platform == "suse" && node.platform_version.to_f >= 12.0
+unless node[:platform_family] == "suse" && node[:platform_version].to_f >= 12.0
```
Comment (vuntz) [inline chef/cookbooks/pacemaker/recipes/default.rb]: Same here.

**Comment by jsuchome** [comment]
@vuntz gating is consistently failing here, but is actually supposed to work? AFAICS no gaiting jobs for HA-related PR's are green...

**Comment by vuntz** [comment]
@jsuchome I'm fine not relying on gating if someone runs a manual mkcloud with this (and cheats by finishing the application of proposals that fail due to some timeout).

**Comment by vuntz** [comment]
On the other hand, the last gating deployed up to ceilometer, and looking at the change, it feels like this means the PR is fine. But I'd still feel more comfortable with a manual run.

**Comment by jsuchome** [comment]
crowbar.c13.cloud.suse.de has this patch applied and most of the stuff deployed...

**Comment by vuntz** [comment]
I also have this on `g9`, with tempest starting in ~10 minutes.

**Comment by vuntz** [comment]
Test passed here. Waiting for results from @jsuchome's tests.

**Comment by jsuchome** [comment]
Everything deployed fine on c13

**Pull request merged**